### DISCUSSION TITLE
UX/UI Enhancement: Get started -> Use eNuts mint flow

### DISCUSSION THREAD

**Discussion by swedishfrenchpress** [cashubtc/eNuts#222] [General]
During the initial device setup, users can choose to use the eNuts mint or add a different one. After selecting the eNuts mint, they are taken to the mint management screen. This doesn't offer the best user experience since they've already decided on adding a mint. It would be more intuitive to redirect users to the "home" page after this action. The home page is a comfortable and familiar environment, whereas transitioning them to mint management settings feels out of place.

![setup-mint-flow](https://github.com/cashubtc/eNuts/assets/78821053/34a57eab-3df9-47d4-b99e-7b2eb8b0d97c)

Just my two sats, open to any feedback as always :).

**Comment by KKA11010** [comment]
Good point and I agree. Luckily, this is a quick change :)

**Comment by KKA11010** [comment]
This will be included in 0.0.4

Closing this discussion now.
